docs/protocol/protocol-reference.mdx

```diff
@@ -0,0 +1,500 @@
+    "chains": [
+      {"chain_id": 1, "token_address": "0xA0b86991..."},
+      {"chain_id": 137, "token_address": "0x2791Bca1..."},
+      {"chain_id": 42161, "token_address": "0xaf88d065..."}
+    ]
+  }
+}
+```
+
+:::tip Asset Discovery
+Use `get_assets` RPC method to retrieve the complete list of supported assets with contract addresses for each <Tooltip content="A blockchain network identifier (uint64)">chain</Tooltip>. Asset support may vary by clearnode.
+:::
+
+---
+
+## EIP References
+
+Ethereum Improvement Proposals referenced or used by the protocol.
+
+### EIP-191: Signed Data Standard
+
+**Status**: Not used in on-chain signatures (chain-agnostic design)  
+**Link**: https://eips.ethereum.org/EIPS/eip-191
+
+**Why not used**: Nitrolite signatures are computed over raw <Tooltip content="Cryptographic hash of a channel state, used for signature verification">stateHash</Tooltip> without EIP-191 prefix to maintain compatibility across different EVM chains and potential non-EVM implementations.
+
+### EIP-712: Typed Structured Data Hashing
+
+**Status**: Not currently used  
+**Link**: https://eips.ethereum.org/EIPS/eip-712
+
+**Future consideration**: May be adopted for improved wallet UX in future versions.
+
+### EIP-1271: Contract Signature Validation
+
+**Status**: Supported by <Tooltip content="A smart contract that validates state transitions according to application-specific rules">adjudicators</Tooltip>  
+**Link**: https://eips.ethereum.org/EIPS/eip-1271
+
+**Usage**: Enables smart contract wallets to sign <Tooltip content="A snapshot of the channel at a point in time, including fund allocations and application-specific data">state</Tooltip> updates as <Tooltip content="An entity (identified by a wallet address) that is part of a channel">participants</Tooltip>.
+
+### EIP-20 (ERC-20): Token Standard
+
+**Status**: Required for all assets  
+**Link**: https://eips.ethereum.org/EIPS/eip-20
+
+**Usage**: All assets must be ERC-20 compliant tokens. The <Tooltip content="The main on-chain contract implementing channel creation, joining, closure, and resizing. It is an implementation of the Nitrolite protocol">Custody Contract</Tooltip> uses `transferFrom` and `transfer` methods.
+
+:::note Standards Compliance
+While the protocol references these EIPs, implementation details may vary. Always consult the specific smart contract code for authoritative behavior.
+:::
+
+---
+
+## Protocol Constants
+
+### Time Constants
+
+```javascript
+DEFAULT_CHALLENGE_PERIOD = 86400  // 24 hours in seconds
+RECOMMENDED_MIN_CHALLENGE = 3600  // 1 hour minimum
+RECOMMENDED_MAX_CHALLENGE = 604800  // 7 days maximum
+```
+
+### Index Constants
+
+```javascript
+CLIENT_IDX = 0       // Client/Creator participant index
+SERVER_IDX = 1       // Server/Clearnode participant index
+PART_NUM = 2         // Number of participants (always 2)
+CLEARNODE_INDEX = 1  // Clearnode's participant index in payment channels
+```
+
+### Version Constants
```

Comment:
We don't have these constants for sure

Reply:
[ff21dbaf86eb4b18cf046b88a721fd96a2e559f6](https://github.com/layer-3/docs/pull/69/commits/ff21dbaf86eb4b18cf046b88a721fd96a2e559f6)